packages/nuxt/src/vite/sourceMaps.ts

```diff
@@ -1,27 +1,39 @@
 import type { Nuxt } from '@nuxt/schema';
 import { type SentryRollupPluginOptions, sentryRollupPlugin } from '@sentry/rollup-plugin';
+import { consoleSandbox } from '@sentry/utils';
 import { type SentryVitePluginOptions, sentryVitePlugin } from '@sentry/vite-plugin';
 import type { NitroConfig } from 'nitropack';
+import type { OutputOptions } from 'rollup';
 import type { SentryNuxtModuleOptions } from '../common/types';
 
 /**
- *  Setup source maps for Sentry inside the Nuxt module during build time (in Vite for Nuxt and Rollup for Nitro).
+ * Whether the user enabled (true, 'hidden', 'inline') or disabled (false) sourcemaps
+ */
+export type UserSourcemapSetting = 'enabled' | 'disabled' | 'unset' | undefined;
+
+/**
+ *  Setup source maps for Sentry inside the Nuxt module during build time (in Vite for Nuxt and Rollup for N itr/**
```

Comment:
```suggestion
 *  Setup source maps for Sentry inside the Nuxt module during build time (in Vite for Nuxt and Rollup for Nitro)
```

Reply:
oh noo that is left from biome (in WebStorm), it continuously messed up this file 😢  Especially in JSDoc and function declarations.